openedx/core/djangoapps/coursegraph/management/commands/tests/test_dump_to_neo4j.py

```diff
@@ -230,6 +230,23 @@ def test_serialize_course(self):
         self.assertEqual(len(nodes), 9)
         self.assertEqual(len(relationships), 7)
 
+    def test_nodes_have_indices(self):
+        """
+        Test that we add index values on nodes
+        """
+        nodes, relationships = self.mss.serialize_course(self.course.id)
+
+        # the html node should have 0 index, and the problem should have 1
+        html_nodes = [node for node in nodes if node['block_type'] == 'html']
+        self.assertEqual(len(html_nodes), 1)
+        problem_nodes = [node for node in nodes if node['block_type'] == 'problem']
+        self.assertEqual(len(problem_nodes), 1)
+        html_node = html_nodes[0]
```

Comment:
my thinking was that this test is pretty tightly coupled to the course that's created in the setUp method. I wanted to be explicit about what we expected from this course in this test in case anything changed about that course